website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,306 @@
+
+- **Create/Delete**: Used when a new entity is produced or destroyed in the system. Example create/delete a project (the project didn’t exist before/will no longer exist).
+- **Add/Remove**: Used when a new relationship or dependency between existing objects is produced or destroyed. Example: Add/Remove a user to/from a project (both the user and the project already existed and will continue to exist, but the relationship itself is established or dissolved).
+
+!!! Do
+
+“Create cluster” or “Delete cluster”
+
+Add user to organization
+
+!!!
+
+!!! Dont
+
+“Create cluster” on a button and “Deploy cluster” on the resulting dialog
+
+Using “remove” and “delete” or “create” and “add” interchangeably
+
+!!!
+
+For actions within an object, use the icon (when possible), the verb and the noun that will be affected.
+
+!!! Do
+
+“Edit project” or “View Project”
+
+![A menu with two actions in it with icons](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-do.png)
+
+!!!
+
+!!! Dont
+
+Miss the object that the user is doing the action. View what?
+
+![A menu with two actions in it, with no icons](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-dont.png)
+
+!!!
+
+When editing an object, use “Save” for the action on the page. When applying another action, use an imperative, present tense verb.
+
+!!! Do
+
+E.g. Edit organization. Action: Save organization
+
+![A modal that has a form to edit an organization](/assets/content/writing-style/writing-style-consistent-termonology-CTA-save-do.png)
+
+E.g. Invite users. Action: Invite users.
+
+![A form to invite users by inputting their emails](/assets/content/writing-style/writing-style-consistent-termonology-CTA-save-do.png)
+
+!!!
+
+!!! Dont
+
+E.g. Edit service principal. Action: Apply changes 
+
+![A form to edit a service principal with the wrong label in the primary button](/assets/content/writing-style/writing-style-consistent-termonology-CTA-apply-dont.png)
+
+!!!
+
+## Punctuation
+
+### Serial/Oxford comma
+
+Use the serial (“Oxford”) comma in lists.
+
+!!! Do
+
+Choose your cluster tier, size, and network.
+
+!!!
+
+!!! Dont
+
+Choose your cluster tier, size and network.
+
+!!!
+
+### Exclamation points
+
+Avoid using exclamation points. Only use it if you’re connecting with understood, strong reader emotion. 
+
+!!! Do
+
+Achieving this centralization is a huge improvement in security posture, but it’s not the end of the journey. This is because applications don't keep secrets! It turns out, most applications do a worse job keeping secrets than our close friends.
+
+!!!
+
+!!! Dont
+
+We are excited to announce Vault 1.3!
+
+!!!
+
+### Percentages
+
+Use the % symbol instead of spelling out "percent."  Correct: 50%
+
+### Ranges and spans
+
+Use a hyphen (-) to indicate a range or span of numbers.
+
+- It takes 20-30 days.
```

Comment:
[blocking] Are we sure this is the right guidance? From some brief Google searching, I believe en dash is actually what we want for numerical ranges. 

From Gemini (with relevant resources quoting APA 2020):
> According to APA 2020 style, a hyphen (-) is used to connect words within a compound adjective, while a dash (either an en dash "–" or an em dash "—") is used to indicate a range of numbers or to set off a parenthetical element within a sentence, with the em dash being longer and signifying a stronger pause than the en dash; essentially, a hyphen is shorter and used for compound words, while a dash is longer and used for interruptions or clarifications within a sentence.

Reply:
Good catch, ill make this update. We should also update our time component and related docs.